Elections/2023/Election-Guidelines.md

```diff
@@ -0,0 +1,80 @@
+* Vote between 25 November 2023 00:00 UTC and 17 December 2023 23:59 UTC via the [voting link](https://vote.heliosvoting.org/helios/e/OpenFeature2023)
+
+# Vacancies
+This election should fill seven seats:
+- fours seats from the existing governance committee
+- three seats from new candidates
+To encourage diversity there will be a maximum of one-third representation on the Governance Committee from any one company at any time. 
+If the outcomes of an election result in greater than 1/3 representation (or maximum of two, whichever is greater), the lowest vote getters from any particular company will be removed until representation on the committee is equal or less than one-third.
+
+
+# Voting process
+
+Anyone can track the 2023 election process via [this GitHub issue](https://github.com/open-feature/community/issues/262).
+We will ensure that all documents and assets related to the 2023 election process are public.
+
+For the 2023 elections, [Helios Voting](https://vote.heliosvoting.org/) was chosen as it's a hosted solution with cryptographic guarantees that no GC members can meddle with the results.
+
+Helios voting also allows us to add GitHub handles to the list of voters in addition to email addresses.
+We need this, as we count contributions based on GitHub contributions and do not always have the contributor's email address.
+The disadvantage of Helios is that it does not support ranked voting.
+
+# Nominations
+
+Anybody is eligible to run for the Governance Committee. During the "call for nomination" period, people can be nominated or nominate themselves by expressing their interest in [this issue](https://github.com/open-feature/community/issues/262) a Pull Request adding said candidate to the [candidates-2023.md](https://github.com/open-feature/community/Elections/2023/Candidates) file in the OpenFeature community repository.
+
+The template in that file includes the following columns:
+
+* Full name
+* GitHub alias
+* Company affiliation (if applicable)
+* Short bio or reasoning to join the Governance Committee (no more than a short paragraph)
+* _Optional_: photo/picture of a nominee
+
+The Pull Request will not be merged until the candidate has confirmed their desire to be nominated (if not self-nominating) and ratified via PR comments.
+
+# Voter Eligibility
+
+All members of the OpenFeature Organization Members, Approvers, Maintainers will automatically be eligible to vote.
+
+# Vote
+
+Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/e/OpenFeature2023) using their GitHub account.
+Voting will be approval voting, where each voter may select up to seven candidates, **four** from the current Governance Committee and three from the new nominees.
+The seven candidates (four from current GC and three from new nominees) with the most votes win the election.
+The winners must be from different organizations, therefore only the highest ranked nominee from each organization will be elected.
```

Comment:
This contrasts with a few lines down where we say 2 from the same org is okay.

Reply:
I agree with Justin. Should we define, that for the same organisation only the candidate with the highest votes will be considered and multiple candidates only fi not all seats fill up?

Reply:
+1

Reply:
updated